pom.xml

```diff
@@ -37,6 +40,7 @@
         <dependency>
             <groupId>org.projectlombok</groupId>
             <artifactId>lombok</artifactId>
+            <version>${lombok.version}</version>
```

Comment:
you don't need to specify lombok's version, as it's inherited from `spring-boot-starter` parent

there you can also find its version in properties

you may remove your `lombok.version` property